src/github.com/getlantern/flashlight/app/mock_autoupdate.go

```diff
@@ -0,0 +1,15 @@
+// +build mockupdate
+
+package app
+
+// This is a public key derived from a private key that is freely available.
+// Don't use in production.
+const packagePublicKey = `-----BEGIN PUBLIC KEY-----
```

Comment:
I'm a bit confused here @xiam . `packagePublicKey` is used nowhere , while `src/github.com/getlantern/autoupdate/keys.go` contains `PackagePublicKey` which is the same as what defined in  `src/github.com/getlantern/flashlight/app/autoupdate.go`.